### PR TITLE
Add og permissions to add/edit/delete own group content.

### PR DESCRIPTION
**Pull request by HelenaEksler**
#19

### CODE REVIEW

**Comment by HelenaEksler** [comment]
@amitaibu I think it's better to add permissions to add/edit own/delete own group content for all group content types. How do you think?

File: shoov/shoov.profile [hunk 1]
```diff
@@ -79,6 +79,23 @@ function shoov_setup_variables() {
  * created.
  */
 function shoov_setup_og_permissions() {
+  $group_content_bundles = og_get_all_group_content_bundle();
+  if (empty($group_content_bundles['node'])) {
```
Comment (amitaibu) [inline shoov/shoov.profile]: this can be removed -- we know there should be group contents

**Comment by HelenaEksler** [comment]
@amitaibu PR is ready.

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request closed without merging**